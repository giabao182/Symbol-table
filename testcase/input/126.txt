INSERT a number
INSERT b string
ASSIGN a 123
ASSIGN b 'abc'
ASSIGN b a
ASSIGN a b