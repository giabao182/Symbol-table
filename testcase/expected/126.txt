success
success
success
success
TypeMismatch: ASSIGN b a
TypeMismatch: ASSIGN a b